backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -958,4 +958,79 @@ databaseChangeLog:
               - column:
                   name: date_tested_backdate
                   type: DATETIME
-                  remarks: timestamp to store a user correction to when the test was ordered
\ No newline at end of file
+                  remarks: timestamp to store a user correction to when the test was ordered
+  - changeSet:
+      id: add-data-hub-upload
+      author: tnielsen2@cdc.gov
+      comment: Add data_hub_upload table used to the automated results uploader.
+      changes:
+        - sql:
+            remarks: Create the enumerations needed for DataHubUpload.
+            sql: |
+              CREATE TYPE ${database.defaultSchemaName}.DATA_HUB_UPLOAD_STATUS as ENUM('IN_PROGRESS', 'SUCCESS', 'FAIL');
```

Comment:
Oh, I thought we were going to leave it as a string in the DB for now, but this is cool. 😄 

Nitpick: `remarks` refers to multiple enumerations, but this migration only has one.